modules/assisted-installer-using-the-assisted-installer.adoc

```diff
@@ -35,6 +35,10 @@ The {ai-full} provides installation functionality as a service. This software-as
 The {ai-full} supports installing {product-title} on premises in a connected environment, including with an optional HTTP/S proxy. It can install the following:
 
 * Highly available {product-title} or Single Node OpenShift (SNO)
+[NOTE]
+====
+Currently, SNO is not supported on {ibmzProductName} and {ibmpowerProductName}.
```

Comment:
You can probably remove the Currently.

Reply:
I think you need `+` characters before and after the admonition.

```
+
[NOTE]
====
Currently, SNO is not supported on {ibmzProductName} and {ibmpowerProductName}.
====
+
```

Reply:
@mburke5678 thanks.

Updated as : 
```
+
[NOTE]
====
SNO is not supported on {ibmzProductName} and {ibmpowerProductName}.
====
+
```